src/App.js

```diff
@@ -0,0 +1,62 @@
+import  Header  from "./components/Header";
+import Cards from "./components/Cards";
+import {useState} from 'react'
+import Footer from "./components/Footer";
+import Instruction from './components/Instruction'
+
+function App() {
+
+  const [cards,setCards] = useState([
+    {
+        id: 1,
+        number: Math.floor(Math.random()*100), 
+    },
+    {
+        id: 2,
+        number: Math.floor(Math.random()*100), 
+    },
+    
+    {
+        id: 3,
+        number: Math.floor(Math.random()*100), 
+    }]) 
+
+//Delete Card
+const deleteCard = (id) =>{
+ setCards(cards.filter((card) => card.id !==id))
```

Comment:
Don't use filter. Remove the selected object from array. Segragate the logic as well